### ISSUE TITLE
Redis Server Url Support 

### ISSUE THREAD

**Issue by Mohamed-Habshey** [senecajs/seneca-redis-pubsub-transport#22]
Hi Everyone

i'm facing a problem here trying to connect to my Redis server (on Heroku) using the Url like this format

```js
[redis:]//[[user][:password@]][host][:port][/db-number][?db=db-number[&password=bar[&option=value]]]
```
so i believe that we need to update the core redis library also this transport plugin needs small change

**Comment by Mohamed-Habshey** [comment]
i had submitted this pr #21  to resolve this issue.

Looking forward for your reply
Regards
M.Habashy

**Comment by Mohamed-Habshey** [comment]
Fixed after merging my pr #21 

Regards
M.Habashy
